File: fs_ruby_on_rails/js_projects/caeser-cipher/caeserCipher.js
const alphaNumeric = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"

const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter word: ", (word) => {
    rl.question("Enter key (must be an integer): ", (key) => {
        key = Number(key);

        console.log(word)
        console.log(key)

        rl.close();
    });
});

File: fs_ruby_on_rails/python/caesercipher/caeser_cipher.py
def shift(key, word):

    result = ""

    for char in word:
        if char in alpha_numeric:
            i = alpha_numeric.index(char)
            new_i = (i + key) % len(alpha_numeric)
            result += alpha_numeric[new_i]      
        else:
            result += char  
    return result

alpha_numeric = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"

word = input("Enter word: ")
key = int(input("Enter key (must be an integer): "))
result = shift(key, word)

print(result)